content/en/docs/tasks/network/extend-service-ip-ranges.md

```diff
@@ -196,3 +200,47 @@ kubectl get servicecidr newcidr1
 ```
 Error from server (NotFound): servicecidrs.networking.k8s.io "newcidr1" not found
 ```
+
+## Kubernetes Service CIDR Policies
+
+Cluster administrators can implement policies to control the creation and
+modification of ServiceCIDR resources within the cluster. This allows for
+centralized management of the IP address ranges used for Services and helps
+prevent unintended or conflicting configurations. Kubernetes provides mechanisms
+like Validating Admission Policies to enforce these rules.
+
+### Preventing Unauthorized ServiceCIDR Creation/Update using Validating Admission Policy
+
+The following example demonstrates how to use a `ValidatingAdmissionPolicy` and
+its binding to restrict the Service CIDR ranges that can be created or updated,
+excluding the default "kubernetes" ServiceCIDR:
```

Comment:
You should explain more clearly exactly what _this_ VAP does. I think it would also be good to provide an example "don't allow any ServiceCIDR changes" VAP, since it's slightly non-obvious (you have to allow the apiserver to create the default one.)

Reply:
added more details and both examples